Maja/script.js

```diff
@@ -0,0 +1,90 @@
+/* 1. 
+2D Array
+Given the 2D Array below, loop through the arrays to print the values. */
+
+let board = [
+  [1, 2, 3],
+  ["quick", "brown", "fox", "jumped", "over", "lazy", "dog"],
+  [true, false]
+];
+for (let i = 0; i <= board.length; i++) {
```

Comment:
Since your index, i, starts from the value 0 then it should iterate < board.length to go until the last element of the array. If you put the sign = then you get an undefined error becuase you are accessing an non-existing arraz element.